proposals/NNNN-GetInputScreen.md

```diff
@@ -0,0 +1,140 @@
+# Get Input Screen
+
+* Author: [Michael Crimando](https://github.com/MichaelCrimando)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite / Web / RPC]
+
+## Introduction
+
+This proposal adds an Input screen for apps. This enables a way for the user to sign in, put in their credit card info, put in their address and more. This is particularly focused at embedded/cloud apps becuase the app lives in the car and the user will still need a way to enter info without having the ability to do so on their phone.
```

Comment:
```suggestion
This proposal adds an Input screen for apps. This enables a way for the user to sign in, put in their credit card information, put in their address and more. This is particularly focused on embedded/cloud apps because the app lives in the car and the user will still need a way to enter information without having the ability to do so on their phone.
```